### PR TITLE
Adding a description for the use_microseconds parameter introduced in MonologBundle v2.11

### PR DESCRIPTION
**Pull request by jorgelbg**
This adds a small paragraph bellow the usage section with an explanation and an example block.

| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | >= 2.3 |
| Fixed tickets | #6466 |

### CODE REVIEW

File: cookbook/logging/monolog.rst [hunk 1]
```diff
@@ -26,6 +26,60 @@ The ``logger`` service has different methods for different logging levels.
...
+                    level="error"
+                />
+            </monolog:config>
+        </container>
+
+    .. code-block:: php
+
+        // app/config/config.php
+        $container->loadFromExtension('monolog', array(
+            'use_microseconds' => false
```
Comment (xabbuh) [inline cookbook/logging/monolog.rst]: missing comma after `false`

File: cookbook/logging/monolog.rst [hunk 2]
```diff
@@ -26,6 +26,60 @@ The ``logger`` service has different methods for different logging levels.
 See :class:`Symfony\\Component\\HttpKernel\\Log\\LoggerInterface` for details
 on which methods are available.
 
+Since version `2.11 of MonologBundle <https://github.com/symfony/monolog-bundle/releases/tag/2.11.0>`
```
Comment (WouterJ) [inline cookbook/logging/monolog.rst]: I don't think we need a link here. Instead, I would use `.. versionadded::`:

```rst
.. versionadded:: 2.11
    The `use_microseconds` option was introduced in MonologBundle 2.11.
```

File: cookbook/logging/monolog.rst [hunk 3]
```diff
@@ -26,6 +26,60 @@ The ``logger`` service has different methods for different logging levels.
...
+an extra parameter for disabling the microsecond precision is available. This is
+kind of a micro optimization but could speed up log generation, specially if you log
+a lot of debug messages.
```
Comment (WouterJ) [inline cookbook/logging/monolog.rst]: can we maybe add a little explanation about what it is disabling and in which cases it should be enabled?

**Comment by WouterJ** [comment]
I would add this in a new section just after "How to Rotate your Log Files" (e.g. "How to Disable Microseconds Precision").

File: cookbook/logging/monolog.rst (hunk 2 above)
Comment (jorgelbg) [inline cookbook/logging/monolog.rst]: Makes more sense this way!

**Comment by jorgelbg** [comment]
I've added a new section with the suggested title

**Comment by WouterJ** [comment]
I like it, good work! :+1:

**Comment by jorgelbg** [comment]
Thanks!!

**Comment by xabbuh** [comment]
👍

**Comment by xabbuh** [comment]
Thank you @jorgelbg.

**Pull request closed without merging**